src/z3c/schema/payments/field.py

```diff
@@ -11,7 +11,7 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""ISO 7812 Credit Card
+"""
```

Comment:
It made the generated HTML for the "Reference" section of the payments page look weird. That one line of text was just sitting there.

It didn't seem to add much value to the module, and pretty much no other modules had a docstring, so the simplest thing was to remove it.

Alternatively, instead of using `.. automodule` I could use `.. autoclass`, but I already had the `.. automodule` template going when I got here and noticed this.